spec/models/user_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe User, {type: 'model'} do
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.